packages/nx/src/command-line/format/format.ts

```diff
@@ -202,3 +203,15 @@ function sortTsConfig() {
     // catch noop
   }
 }
+
+function getPrettierPath() {
+  let prettierVersion: string;
+  try {
+    prettierVersion = require(require.resolve('prettier/package.json')).version;
```

Comment:
Can you use `readModulePackageJson` for this? Its exported from 'nx/src/utils/package-json'

Reply:
Done :+1: